py_irt/models/two_param_logistic.py

```diff
@@ -237,23 +233,6 @@ def get_guide(self):
         else:
             return self.guide_hierarchical
 
-    def fit(self, models, items, responses, num_epochs):
```

Comment:
Just to confirm, this is being removed because we have the model fitting code in the trainer, correct? If so, should we also drop .fit() from the 1PL model?

Reply:
Yup, I happened to be updating this model, I can remove the fit from the 1pl in followup PR